source/gameengine/GameLogic/SCA_PropertySensor.cpp

```diff
@@ -113,6 +113,9 @@ SCA_PropertySensor::~SCA_PropertySensor()
 
 bool SCA_PropertySensor::Evaluate()
 {
+	if (!(strlen(m_checkpropval.c_str()))) {
```

Comment:
You could use std::string, also i think the better place is in CheckPropertyCondition because this function is the only one reading the property string (as far as i see).